dojo/risk_acceptance/helper.py

```diff
@@ -112,9 +113,20 @@ def remove_finding_from_risk_acceptance(risk_acceptance, finding):
     finding.save(dedupe_option=False)
     # best effort jira integration, no status changes
     post_jira_comments(risk_acceptance, [finding], unaccepted_message_creator)
+    # Add a note to reflect that the finding was removed from the risk acceptance
+    if user is not None:
+        finding.notes.add(Notes.objects.create(
+            entry=(
+                f"{Dojo_User.generate_full_name(user)} removed this finding from the risk acceptance: "
```

Comment:
Can we also include the user ID in parenthesis after the full user name?

Users can change their names so this isn't always a definitive value - user ID won't change so is the safest way to identify who did the action though not user friendly. Having both is optimal.

This way I can't change my DefectDojo user data to "Cody Maffucci" and get you in trouble :wink:

Reply:
Sure, can do. Please note that the if the user object changes in any way, the note written on the finding will not change. The note itself can be edited, but the modification will be auditlogged on the note object